sql/core/src/main/java/org/apache/spark/sql/streaming/Trigger.java

```diff
@@ -89,15 +90,23 @@ public static Trigger ProcessingTime(String interval) {
   }
 
   /**
-   * A trigger that process only one batch of data in a streaming query then terminates
-   * the query.
+   * A trigger that processes all available data in a single batch then terminates the query.
```

Comment:
Maybe add "For better scalability,  `AvailableNow` can be used alternatively to process it in multiple batches.".